EXAMPLES.md

```diff
@@ -0,0 +1,5 @@
+## Example Apps
+
+* **[Calculator](https://ahfarmer.github.io/calculator/)** This is a full-screen, mobile-friendly calculator app similar to the iOS calculator:  [view code in on the GitHub repository](https://github.com/ahfarmer/calculator).
+
+* **[Emoji Search](https://ahfarmer.github.io/emoji-search/)** This app is for searching emoji. [view code in on the GitHub repository](https://github.com/ahfarmer/emoji-search).
```

Comment:
Let's add something like "Did you build a high-quality open source example using Create React App? Edit this page to add it."